app/controllers/home_controller.rb

```diff
@@ -1,5 +1,9 @@
 class HomeController < ApplicationController
   def index
     @poems = Poem.all.order(id: :desc).includes(:user)
+    respond_to do |format|
+      format.html
+      format.json { render template: "poems/index" }
```

Comment:
__[rails_best_practices]__
* simplify render in controllers [:ref](http://rails-bestpractices.com/posts/2010/12/12/simplify-render-in-controllers/)